app/controllers/display_controller.rb

```diff
@@ -75,6 +75,48 @@ def read_all_works
     session[:col_id] = @collection.slug
   end
 
+  def display_page
```

Comment:
Consider whether we want to display this for all pages, or only pages that have a status indicating that they have content.